hera_cal/abscal.py

```diff
@@ -118,17 +117,22 @@ def __init__(self, model, data, wgts=None, antpos=None, freqs=None, pol_select=N
         pol_select : list of polarizations you want to keep in data
                      type=list, dtype=str, Ex. ['xx', 'yy']
         """
+        # set pols to None
         pols = None
+
         # check format of model
```

Comment:
Clarify in comments that this is for when the model is not a dictionary already.